doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2809,22 +2812,29 @@ Configure a {product-title} instance to act as a remote copy where data from the
 [[configuring-the-global-copy]]
 ===== Configuring the Global Copy
 
-Configure a {product-title} instance to act as the global copy from which data is replicated to remote copies. This procedure must only be performed on servers where the *Database Synchronization* role will be enabled. These settings must be completed before enabling that role.
+Configure a {product-title} instance to act as the global copy to which data is replicated from the remote copies.
 
-. Navigate to menu:Settings[Configuration].
+. Navigate to the user settings in the top-right menu.
+. Click *Configuration*.
 . Click the *Settings* accordion.
 . Click the region where the instance is located.
 . Click *Replication*.
 . Select *Global* from the *Type* list.
-. Add a subscription:
-.. Click *Add Subscrption*.
+. Click *Add Subscription*.
+image:centralized_admin_disabled.png[Centralized Administration Config]
 .. Enter the name of the database on the remote copy in the *Database* field.
 .. Enter the IP address or fully qualified domain name of the remote copy in the *Host* field.
 .. Enter the name of a database user able to access the database in the *Username* field.
 .. Enter the password of the database user in the *Password* field.
 .. Enter the port by which the database is accessed in the *Port* field.
+.. In *Actions*, click the menu on the right to see a list of actions, including *Enable Central Admin*, *Validate*, and *Delete*. You can also *Update* your existing subscription.
+* Click *Enable Central Admin*. Note that central administration can only be enabled after the initial data sync is complete. So you must save the subscription before this can be done. 
+* If *Central Admin Enabled* shows _Yes_, that is if centralized administration is currently enabled, you can *Disable Central Admin* or *Re-Enable Central Admin*. You can also *Validate* or *Delete* your subscription.
+image:centralized_admin_enabled.png[Centralized Administration Enabled]
```

Comment:
@carbonin I agree, thanks. I've made a few edits to the existing section. Also added a separate section about enabling central admin after adding the subscription. Let me know what you think. Thanks.